samples/bluetooth/mesh/light/child_image/hci_rpmsg.conf

```diff
@@ -0,0 +1,10 @@
+# Copy controller configuration from prj.conf
+CONFIG_BT_BUF_ACL_RX_SIZE=73
```

Comment:
```suggestion
CONFIG_BT_BUF_ACL_RX_SIZE=37
```
here and below

Reply:
No, check with nrf52840. Default for mesh is 73 if GATT is enabled

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/8959b838bf15118dadc12c21e3d4ca5a92039c77/subsys/bluetooth/common/Kconfig#L48-L53

Reply:
It is a bit expensive to keep 73 bytes just to not fragment public key exchange that happens very rarely (once per provisioning procedure). It is not blocker, but I'd recommend revising this digit. Nothing critical if public key exchange will be fragmented.

Reply:
It is as it is. Proper fix will be done as part of NCSDK-21755

Reply:
Currently, the values are aligned with what the default to if sample is build for nRF52840.